CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.0] - 2021-08-02
+### Added
+- Added a base driver class mechaspin::parakeet::Driver which holds common functionality between sensor drivers
+- Added a driver class to support the Parakeet ProE (mechaspin::parakeet::ProE::Driver)
+- Added an example application to set the IPv4 settings of a Parakeet-ProE sensor
+
+### Modified
+- Updated SimpleExample to support the Parakeet ProE sensor
```

Comment:
I kinda like having the changes to the SimpleApplication listed here and having just one CHANGELOG, maybe we should indicate here the version of the SimpleApplication? Something like:

Updated SimpleExample to support the Parakeet ProE sensor (v2.0.0)?